src/sources/socket/mod.rs

```diff
@@ -152,12 +151,12 @@ impl SourceConfig for SocketConfig {
             }
             Mode::Udp(config) => {
                 let log_namespace = cx.log_namespace(config.log_namespace);
-                let decoder = DecodingConfig::new(
-                    config.framing().clone(),
-                    config.decoding().clone(),
-                    log_namespace,
-                )
-                .build()?;
+                let decoding = config.decoding().clone();
+                let framing = config
```

Comment:
Now the `framing` depends on the `decoding` that's being used, so the chunked gelf framing is used as default when using the gelf deserializer